tests/test_subprocess.py

```diff
@@ -13,6 +14,20 @@
 
 PYTHON = sys.executable
 
+win_path_skip = pytest.mark.skipif(
+    sys.platform.startswith("win") and sys.version_info < (3, 8),
+    reason="Path in subprocess not supported before 3.8 on Windows",
```

Comment:
Instead of skipping it on Windows for 3.7, I'd rather prefer to behave similarly to the real subprocess on this platform. I see it raises `TypeError` there, so I think this plugin should recreate this behavior to be reliable.